microcloud/cmd/microcloud/main_init.go

```diff
@@ -414,6 +419,68 @@ func AddPeers(sh *service.Handler, systems map[string]InitSystem) error {
 	return nil
 }
 
+func validateSystems(s *service.Handler, systems map[string]InitSystem) (err error) {
+	curSystem, bootstrap := systems[s.Name]
+	if !bootstrap {
+		return nil
+	}
+
+	// Assume that the UPLINK network on each system is the same, so grab just
+	// the gateways from the current node's UPLINK to verify against the other
+	// systems' management net addrs
+	hasIP4Gateway, hasIP6Gateway := false, false
+	var ip4Gateway, ip6Gateway string
+	var ip4GatewayNet, ip6GatewayNet *net.IPNet
+
+	for _, network := range curSystem.Networks {
+		if network.Type == "physical" && network.Name == "UPLINK" {
+			ip4Gateway, hasIP4Gateway = network.Config["ipv4.gateway"]
+			ip6Gateway, hasIP6Gateway = network.Config["ipv6.gateway"]
+			ovnRanges, hasOVNRanges := network.Config["ipv4.ovn.ranges"]
```

Comment:
https://documentation.ubuntu.com/lxd/en/latest/reference/network_physical/#network-physical-network-conf:ipv6.ovn.ranges is a thing in LXD, does microcloud offer to support it, if so we need to also check it here.

Reply:
No, so far MicroCloud only offers a range for ipv4. Perhaps this is something we should add (but out of scope for this pr)

Reply:
should we add it here just in case it gets added in the future and missed ?